plugin/src/main/java/io/jenkins/plugins/casc/TokenReloadAction.java

```diff
@@ -0,0 +1,90 @@
+package io.jenkins.plugins.casc;
+
+import com.google.common.base.Strings;
+import hudson.Extension;
+import hudson.model.UnprotectedRootAction;
+import hudson.util.Scrambler;
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.annotation.CheckForNull;
+import javax.servlet.http.HttpServletRequest;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.interceptor.RequirePOST;
+
+/**
+ * If enabled by setting environment variable "CASC_ALLOW_LOCAL_RELOAD" to "true" it allows reloading
+ * configuration by issuing a POST request to "http://localhost:8080/reload-configuration-as-code/".
+ */
+@Extension
+public class TokenReloadAction implements UnprotectedRootAction {
+    public static final Logger LOGGER = Logger.getLogger(TokenReloadAction.class.getName());
+
+    public static final String URL_NAME = "/reload-configuration-as-code";
+    public static final String RELOAD_TOKEN = "jcasc.reloadToken";
```

Comment:
This doesn't fix with the current namespace:
https://github.com/jenkinsci/configuration-as-code-plugin/blob/b15e99f95d65137cb8d8903b4134ac6589888cc9/plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationAsCode.java#L102
> casc.jenkins.config

Any of the following should be fine:

```suggestion
    public static final String RELOAD_TOKEN = "casc.reload.token";
```

```suggestion
    public static final String RELOAD_TOKEN = "casc.jenkins.reload-token";
```

```suggestion
    public static final String RELOAD_TOKEN = "casc.jenkins.reload.token";
```

Reply:
Also this is a property not the token
```suggestion
    public static final String RELOAD_TOKEN_PROPERTY = "jcasc.reloadToken";
```